pkgs/top-level/all-packages.nix

```diff
@@ -20871,8 +20871,8 @@ with pkgs;
 
   openldap = callPackage ../development/libraries/openldap { };
 
-  opencolorio = callPackage ../development/libraries/opencolorio {
-    inherit (darwin.apple_sdk.frameworks) Carbon GLUT Cocoa;
+  opencolorio = darwin.apple_sdk_11_0.callPackage ../development/libraries/opencolorio {
```

Comment:
Should `darwin.apple_sdk_11_0.callPackage` be used if the derivation also supports Linux?

Reply:
See other uses in `all-packages.nix` - apparently yes.